### PR TITLE
Update home page content, round two, per @davisagli

### LINKED ISSUE #85: Default homepage content updates round 2
@stevepiercy I would change the description too; it doesn't make sense for a demo site

_Originally posted by @davisagli in https://github.com/plone/plone.volto/pull/84#discussion_r956240351_

### PR DESCRIPTION
**Pull request by stevepiercy**
Ran `make black` for formatting.

Closes #85

### CODE REVIEW

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
## Preview
Screen Shot 2022-08-30 at 4 27 14 PM

File: news/85.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Update default homepage content for Slate, round 2. @stevepiercy
```
Comment (davisagli) [inline news/85.bugfix:1]:
```suggestion
Update demo homepage content for Slate, round 2. @stevepiercy
```

**Review by davisagli — APPROVED** [review]
Thanks!

**Comment by davisagli** [comment]
@stevepiercy most of the github actions checks were not triggered since this was opened from your fork. I pushed the existing commit to plone/plone.volto to trigger them

**Comment by stevepiercy** [comment]
@davisagli thanks for merging. For some unknown reason, I still do not have access to create a new branch in this repo, which is why I had to do it from my fork. If you can flip a switch to allow me access, I would appreciate it.

**Comment by davisagli** [comment]
@stevepiercy That's odd! I can see that you're in the Developers team of the plone organization, which has write access to the repo. Are you trying to create a branch through the web, or by pushing?

**Comment by stevepiercy** [comment]
To close the loop, the fix was to clear out my macOS Keychain of all `github` items, per https://stackoverflow.com/a/47466980/2214933

**Pull request merged**